README.md

```diff
@@ -338,6 +345,23 @@ The options `-h`, `-i`, `-r`, `-f` are exclusive. The options `-s` and `-k` are
 at the LPC (where each interactive node has its own scheduler). The script can ssh to each node and run itself to modify the jobs
 on that node (because each scheduler can only be accessed for write operations from its respective node).
 
+The option `-X` will resubmit the jobs (just like `-s`), except that it will tell the job to get its input file from a specific site based on the list of sites where that file
+is located and some user preferences. Therefore, the options `-X` and `-s` are exclusive, as are the options `-X` and `-k`. The options `-C`, `-D`, `-K`, `-L`, `-U`, and `-V`
+are only applicable when using the option `-X`. There are two ways in which the program can find the appropriate input file(s) for the job:
+1. specify an HTCondor ClassAd name, where the ClassAd contains a comma separated list of files (`-C`).
+2. specify the location of a set of log files and a key with which to parse the logs files (`-L`/`-K`). It is expected that the log will contain a single line with a comma separated list of files.
+
+This is similar in spirit to using the `-x` option, except in that case all of the resubmitted jobs will try to access the input files from a user specified location. In this case,
```

Comment:
I find the phrasing here a little confusing. Suggestion:
The `-x` and `-X` options are similar. If just the `-x` option is used, all of the resubmitted jobs will try to access the input files from a user-specified location. If just the `-X` option is used, the user doesn't specify a particular location, but a set of loose preferences (see below). If both the `-X` and `-x` options are used, then the jobs will preferentially read their input from the site found automatically, with the redirector or site name given by `-x` as a fallback if no suitable site is found.